intranet/utils/helpers.py

```diff
@@ -162,7 +162,10 @@ def dark_mode_enabled(request):
     if request.GET.get("dark", None):
         return request.GET["dark"] in ["1", "True"]
 
-    if request.user.is_authenticated and request.user.dark_mode_properties.dark_mode_unlocked:
-        return request.user.dark_mode_properties.dark_mode_enabled
+    if request.user.is_authenticated:
+        if request.user.dark_mode_properties.dark_mode_unlocked:
```

Comment:
You could just return dark_mode_unlocked and dark_mode_enabled. This would also allow you to get rid of the `return False` at the end.